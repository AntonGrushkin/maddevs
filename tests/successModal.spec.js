import {
  mount
} from '@vue/test-utils';

import SuccessModal from '@/components/Modals/success-modal';

describe('Success modal', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = mount(SuccessModal, {
      stubs: ['modal']
    });
  });

  // ------ IMPORTANT ----- //
  test('is Vue\'s instance', () => {
    expect(wrapper.isVueInstance()).toBeTruthy();
  });

  test('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot();
  });
  // --------------------- //
});
