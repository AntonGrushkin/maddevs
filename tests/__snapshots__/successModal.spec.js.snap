// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Success modal renders correctly 1`] = `
<modal-stub
  class="success-modal"
  name="success-modal"
>
  <img
    alt="Close modal"
    class="close-modal"
    src="@/assets/img/common/close-icon.svg"
  />
   
  <div
    class="success-modal__conten-wrapper"
  >
    <img
      alt="Hand"
      src="@/assets/img/common/success-modal-hand.svg"
    />
     
    <div
      class="success-modal__text-content"
    >
      <div
        class="success-modal__title"
      >
        Thanks for your 
        <br />
         interest in Mad Devs
      </div>
       
      <div
        class="success-modal__info-text"
      >
        The form has been submitted. 
        <br />
         We will contact you as soon as possible.
      </div>
    </div>
  </div>
</modal-stub>
`;
